File: Factorial.java
import java.util.Scanner;

public class Factorial{
public static void main(String... args){
Scanner scanner = new Scanner(System.in);

	System.out.println("enter a number");
	int number = scanner.nextInt();

	long result = Kata.factorial(number);
	System.out.print(result);

}

}

File: factorial.c
# include <stdio.h>


int main() {
    int x;
    int result = 1;

 
	printf("Enter the dividend: ");
    	scanf("%d", &x);

	for (int i = 1; i <= x; i++){

	result *= i;
}
    


        
        printf("%d", result);
	


    return 0;
}
